source/GUI/Form1.cs

```diff
@@ -488,5 +500,36 @@ private void timersButton_Click(object sender, EventArgs e)
             _parcer.Storage.TimeSMVType = timetypeTextBox.Text;
             _parcer.Storage.Tmax = Convert.ToInt32(tmaxTextBox.Text);
         }
+
+        private void updateButton_Click(object sender, EventArgs e)
+        {
+            /*
+            _selectedVariable.Name = varArraySizeTextBox.Text;
+            foreach (Variable variable in _connectedVars)
+            {
+                variable.Name = _selectedVariable.Name;
+            }
+            variablesTreeView.SelectedNode.Name = _selectedVariable.Name;
+            _parcer.Storage.Variables.FirstOrDefault(v => v.FBType == _selectedFbType && v.Name == varArraySizeTextBox.Text).Name = _selectedVariable.Name;
+            */
+            if (Smv.DataTypes.IsNumberType(_selectedVariable.SmvType))
+            {      
+                int lb = Convert.ToInt32(lowerLimitTextBox.Text);
+                int ub = Convert.ToInt32(upperLimitTextBox.Text);
+                if (lb < ub)
+                {
+                    _selectedVariable.SmvType = new Smv.DataTypes.RangeSmvType(lb, ub);
+                    foreach (Variable variable in _connectedVars)
+                    {
+                        variable.SmvType = new Smv.DataTypes.RangeSmvType((Smv.DataTypes.RangeSmvType)_selectedVariable.SmvType);
+                    }
+                }
+                else
+                {
+                    Program.ErrorMessage("Invalid range type!");
+                    return;
+                }
+            }
+        }
```

Comment:
Added "Lower bound" and "Upper bound" (LL,UL) value modification  via Update button.